examples/GatewayESP8266OTA/GatewayESP8266OTA.ino

```diff
@@ -133,32 +131,33 @@
 #include <ESP8266WiFi.h>
 #endif
 
+#include <ArduinoOTA.h>
 #include <MySensors.h>
 
 void setup()
 {
 	// Setup locally attached sensors
 	ArduinoOTA.onStart([]() {
-		debug("ArduinoOTA start\n");
+		Serial.println("ArduinoOTA start");
```

Comment:
How come Serial.println is used here instead of GATEWAY_DEBUG ?

Reply:
GATEWAY_DEBUG should not be used in examples